File: Assets/Plugins/iOS/PlayPhoneMultiNet/core/ws/MNWSSessionSignedClientToken.m
//
//  MNWSSessionSignedClientToken.m
//  MultiNet client
//
//  Copyright 2011 PlayPhone. All rights reserved.
//


#import "MNWSSessionSignedClientToken.h"

@implementation MNWSSessionSignedClientToken

-(NSString*) getClientTokenBody {
    return [self getValueByName :@"client_token_body"];
}

-(NSString*) getClientTokenSign {
    return [self getValueByName :@"client_token_sign"];
}


@end

File: Assets/Plugins/Android/src/com/playphone/multinet/core/ws/data/MNWSSessionSignedClientToken.java
//
//  MNWSSessionSignedClientToken.java
//  MultiNet client
//
//  Copyright 2011 PlayPhone. All rights reserved.
//

package com.playphone.multinet.core.ws.data;

public class MNWSSessionSignedClientToken extends MNWSGenericItem
 {
  public String getClientTokenBody ()
   {
    return getValueByName("client_token_body");
   }

  public String getClientTokenSign ()
   {
    return getValueByName("client_token_sign");
   }
 }
